src/mmcmdl.c

```diff
@@ -1770,6 +1775,7 @@ static flag getFullArg(long arg, const char *cmdList1) {
       if (!tmpFp) {
         let(&tmpStr,  cat("?Sorry, couldn't open the file \"", tmpStr, "\".", NULL));
         printCommandError(errorLine, arg, tmpStr);
+        free_vstring(tmpStr);
```

Comment:
[move free of temps to just after last use]
Is there a good reason why free_vstring is applied to tmpStr here, but not to tmpArg, assigned in line 1742?  It is difficult for me to keep track of the stack.

Reply:
No, you are right. This refactor has itself gone through some refactorings and that one seems to have been lost in the shuffle. It's supposed to be dropped on line 1746 just before it goes out of scope.